### PR TITLE
[Test] Add unit tests for HDR/TDigest PercentilesAggregators

### PR DESCRIPTION
**Pull request by tlrx**
Relates to #22278

### CODE REVIEW

File: core/src/test/java/org/elasticsearch/search/aggregations/metrics/percentiles/tdigest/TDigestPercentilesAggregatorTests.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        });
+
+
+
+        XContentBuilder xContentBuilder = XContentFactory.contentBuilder(XContentType.JSON);
+        xContentBuilder.startArray();
+        xContentBuilder.rawValue(new BytesArray("\"value1\""), XContentType.JSON);
+        xContentBuilder.rawValue(new BytesArray("\"value2\""), XContentType.JSON);
+        xContentBuilder.endArray();
+        String hson = xContentBuilder.bytes().utf8ToString();
```
Comment (cbuescher) [inline core/src/test/java/org/elasticsearch/search/aggregations/metrics/percentiles/tdigest/TDigestPercentilesAggregatorTests.java]: I think I've seen this somewhere else today ;-)

**Review by cbuescher — APPROVED** [review]
LGTM, left one comment where I think there was something left over...

File: core/src/test/java/org/elasticsearch/search/aggregations/metrics/percentiles/tdigest/TDigestPercentilesAggregatorTests.java (hunk 1 above)
Comment (tlrx) [inline core/src/test/java/org/elasticsearch/search/aggregations/metrics/percentiles/tdigest/TDigestPercentilesAggregatorTests.java]: Arf...

**Comment by tlrx** [comment]
Thanks @cbuescher

**Pull request merged**